scalability_and_performance/scaling-cluster-monitoring-operator.adoc

```diff
@@ -5,11 +5,11 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-{product-title} exposes metrics that the Cluster Monitoring Operator collects and stores in the Prometheus-based monitoring stack. As an administrator, you can view system resources, containers and components metrics in one dashboard interface, Grafana.
+{product-title} exposes metrics that the Cluster Monitoring Operator collects and stores in the Prometheus-based monitoring stack. As an administrator, you can view system resources, containers and components metrics in the {product-title} web console in *Monitoring* > *Dashboards*.
```

Comment:
should be in *Observe* > *Dashboards*.

Reply:
same for https://deploy-preview-38687--osdocs.netlify.app/openshift-enterprise/latest/logging/cluster-logging-dashboards.html#cluster-logging-dashboards-access_cluster-logging-dashboards